handler_success.go

```diff
@@ -68,6 +69,16 @@ func tagHeaders(r *http.Request, th []string, tags []string) []string {
 	return tags
 }
 
+func addVersionHeader(w http.ResponseWriter, r *http.Request) {
+	if ctxGetDefaultVersion(r) {
+		if vinfo := ctxGetVersionInfo(r); vinfo != nil {
+			if config.Global.VersionHeader {
+				w.Header().Set("X-Tyk-Default-Version", vinfo.Name)
```

Comment:
You forgot to update "X-Tyk-Default-Version" to `config.Global.VersionHeader`

Reply:
`config.Global.VersionHeader` is just a boolean flag for whether the header should be added or not.

Reply:
Right, I just thought you initially wanted make it configurable

Reply:
I just intended it to be on-or-off depending on whether the user wanted to expose their configuration. If you think having it be a configurable string is better I can make that change easily.

Reply:
Yeah, let's make it configurable, update bool to string.